test/integration/test_integration_basics.py

```diff
@@ -30,7 +30,7 @@ def test_http_h1(http_test_server_fixture):
   assertCounterEqual(counters, "upstream_cx_rx_bytes_total", 3400)
   assertCounterEqual(counters, "upstream_cx_total", 1)
   assertCounterEqual(counters, "upstream_cx_tx_bytes_total",
-                     1400 if http_test_server_fixture.ip_version == IpVersion.IPV6 else 1500)
+                     1400 if http_test_server_fixture.ip_version == IpVersion.IPV6 else 1450)
```

Comment:
The two literals (1400, 1450) would benefit from being well named constants with some explanation.

Reply:
The change here is triggered by that we now connect to 0.0.0.0 instead of 127.0.0.1, resulting in a decrease of `upstream_cx_tx_bytes_total` as the request header gets slightly smaller.

Reply:
After discussing on Slack, we concluded this can stay as is; it's not as if this has anything to do with MTU sizes :-)